akka-docs/rst/scala/code/docs/http/scaladsl/server/directives/DebuggingDirectiveExtendedExample.scala

```diff
@@ -0,0 +1,63 @@
+package com.knoldus.logging
+
+/**
```

Comment:
this header should go and instead the standard Lightbend Copyright header should be included.